### PR TITLE
Fixed torch.exp inf error(measurements_loss)

### PR DESCRIPTION
**Pull request by woo1**
- When I tried with weight over 150kg, measurements_loss became inf. So I couldn't fit with the person.
- I am not sure this is the best way to fix. But when I tried another method like [log1pexp](https://stackoverflow.com/questions/60903821/how-to-prevent-inf-while-working-with-exponential) it doesn't have an effect
- test data : https://drive.google.com/file/d/1Tea9d5VPuD0LTq19mPJlCm-MSgIlJyi1/view?usp=sharing

### CODE REVIEW

**Comment by muelea** [comment]
Thanks for pointing this out. A flag that people can set, e.g. --weight_above_150, would be more appropriate than changing the actual loss term.

**Comment by woo1** [comment]
Thanks for your feedback.
I updated the code to use args(--weight_above_150).
